CreationalDesignPatterns/AbstractFactory/CoffeeFactory/Macchiato.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace CoffeeFactory
+{
+    public class Macchiato : MilkCoffee

```

Comment:
Can we use lambda function and extract all the magic numbers and strings in constants